zawata/S19-Team2
# Flask server

Added configuration and get_objects method.
To test:
1. create `S19-Team2/config/kernels` directory
2. in `config/kernels`, add LMAP_FullTrajectory.bsp (From team drive)
3. in `config` create a file `config.json` with the following contents:
`{
  "main_subject": -39,
  "kernels": [
    "LMAP_FullTrajectory.bsp"
  ]
}`
4. Make a get request to `http://localhost:5000/api/all_objects`; should return a list of objects with ids in them.
5. Make a get request to `http://localhost:5000/api/all_objects?time=600000000` 
- You should recieve an array of json objects containing position/velocity information and ids.
